### ISSUE TITLE
Running in release mode for Windows breaks icon support

### ISSUE THREAD

**Issue by Bombfuse** [not-fl3/miniquad#264]
While running the Instancing example

debug mode
![image](https://user-images.githubusercontent.com/26308406/151090995-a752acbb-2234-4d5c-8e84-acab0b2a9e87.png)

release mode
![image](https://user-images.githubusercontent.com/26308406/151091002-54ca9a6f-131d-4313-9c3f-f1b8748046d1.png)

**Comment by pellico** [comment]
In my code I set the icon.
```rust let conf =  Conf {
        window_title: "KTanks".to_owned(),
        window_width: 1024,
        window_height: 768,
        icon : Some(ICON),
        ..Default::default()
    };
```
 In debug mode it is working perfectly.
in release mode:
![image](https://user-images.githubusercontent.com/16132532/154806109-6a197e15-0eed-4c45-be05-5929b7cc5de8.png)
.
If I declare all fields like below
```rust 
let conf =  Conf {
        window_title: "KTanks".to_owned(),
        window_width: 1024,
        window_height: 768,
        icon : Some(ICON),
        high_dpi: false,
        fullscreen: false,
        sample_count: 1,
        window_resizable: true,

    };
```
in release mode the icon get even more corrupted:
![image](https://user-images.githubusercontent.com/16132532/154806212-2cba6695-12f6-4cb0-a006-7917851d9be9.png)

**Setup**

- Windows 10 64 bit
- Macroquad v 0.3.13 -->  miniquad v0.3.0-alpha.42
- stable-x86_64-pc-windows-msvc (default)
- rustc 1.58.1 (db9d1b20b 2022-01-20)

**Comment by not-fl3** [comment]
Should be fixed in miniquad-v0.3.1
